lambda-on-binaris/function.js

```diff
@@ -0,0 +1,41 @@
+const { join } = require('path');
+const { execute } = require('lambda-local');
+
+function makeAPIGatewayEvent(body, context) {
+  const isBase64Encoded = (!body || body === {}) ? true : false;
```

Comment:
i'm almost certain `!body` is a bug.
for example if `body === false`
moreso, isBase64Encoded is probably not correct either since 1. nothing about the encoding is suggested here and 2. in lambda the body is never parsed